src/app/dashboard/dashboard.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DashboardComponent } from './components/dashboard.component';
+import { TranslateModule } from '@ngx-translate/core';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  declarations: [
+    DashboardComponent
+  ],
+  imports: [
+    CommonModule,
+    TranslateModule,
+    RouterModule.forChild([{ path: '', component: DashboardComponent }])
```

Comment:
What I don't like is that routes are not declared in one way. If we have already used a routing module (LayoutRoutingModule) we should do it in the same way also here.